command/secrets/generate/params.go

```diff
@@ -44,9 +45,6 @@ type generateParams struct {
 
 func (p *generateParams) getRequiredFlags() []string {
 	return []string{
-		dirFlag,
-		tokenFlag,
-		serverURLFlag,
```

Comment:
These flags are actually required for the base file system secrets generation.

You should add a check when validating the flags to make sure they're present if the mode is disk mode.

Reply:
Please correct me if I'm wrong, but I don't think we use `secrets generate` command when we want to use local FS. The command we use is just `secrets init`.

Reply:
You're completely right, I've overlooked this 🤦 

Resolving